packages/components/organisms/f-checkout/stories/Checkout.stories.js

```diff
@@ -44,7 +44,7 @@ export const CheckoutComponent = () => ({
             default: text('Checkout ID', checkoutId)
         },
         checkoutUrl: {
-            default: select('Checkout Url', [deliveryUrl, collectionUrl], deliveryUrl)
+            default: select('Checkout Url', [deliveryUrl, collectionUrl, 'An invalid URL'], deliveryUrl)
```

Comment:
Is there a reason this isn't called 'invalidUrl' to match the others? teeny but would be nice if they were the same format 😄

Reply:
This one was a hardcoded text, whereas the other ones are JS variables. I'm happy to extract it into a variable, if people prefer :)

Reply:
ahh okay cool, that makes sense! Probably no need to change it then! :)